.github/workflows/job_build_cid.yml

```diff
@@ -243,9 +293,9 @@ jobs:
         if: ${{ !steps.cache-restore.outputs.cache-hit }}
         run: |
           cmake \
-            --build "${OPENVINO_BUILD_DIR}" \
+            --build ${OPENVINO_BUILD_DIR} \
             --parallel \
-            --config "${CMAKE_BUILD_TYPE}" \
+            --config ${CMAKE_BUILD_TYPE} \
```

Comment:
what is the reason of removing quotes here? don't they help to handle special chars and spaces better?

Reply:
Merge artifact. Doesn't affect the flow though. I might return it back within the next PRs for higher stability